cglims/cli/commands.py

```diff
@@ -194,17 +194,20 @@ def set_defaults(lims_sample):
     lims_sample.udf['Index number'] = 'NA'
     lims_sample.udf['Sample Buffer'] = 'NA'
     lims_sample.udf['Reference Genome Microbial'] = 'NA'
-    if lims_sample.udf['Gender']:
-        lims_sample.udf['Gender'] = lims_sample.udf['Gender'].upper()
-    if lims_sample.udf['Status']:
-        lims_sample.udf['Status'] = lims_sample.udf['Status'].lower()
+    gender = lims_sample.udf.get('Gender')
+    if gender:
+        lims_sample.udf['Gender'] = gender
```

Comment:
aren't you just replacing the value by itself here? hehe
missing the `.lower()` call?